stable/test/integration/_exec.pony

```diff
@@ -0,0 +1,85 @@
+use "ponytest"
+use "files"
+use "process"
+use "regex"
+
+actor _Exec
+  new create(
+    h: TestHelper,
+    cmdline: String,
+    notifier: ProcessNotify iso)
+  =>
+    try
+      let args = cmdline.split_by(" ")
+      let path = FilePath(h.env.root as AmbientAuth, "build/release/stable")?
```

Comment:
this will not work if stable was compiled with `debug` config.
What you could do is hand over the `config` var to the ponyc process as user define (with `--define config_release` or `--define config_debug`) and use an ifdef here on the value as string to chose the right binary:

```pony
ifdef "config_release" then
  let stable_binary = "build/release/stable"
else
  let stable_binary = "build/debug/stable"
end
```

something along those lines.